File: server/src/routes/health.ts
import { Router } from 'express';

const router = Router();

router.get('/', (req, res) => {
  res.json({
    success: true,
    status: 'healthy',
    timestamp: new Date().toISOString(),
    service: 'SkillProof API',
    version: '1.0.0'
  });
});

export { router as healthRouter };

File: backend/app/routes/health.py
from fastapi import APIRouter
from datetime import datetime

router = APIRouter()


@router.get("/")
async def health_check():
    return {
        "success": True,
        "status": "ok",
        "timestamp": datetime.utcnow().isoformat(),
        "service": "skillproof-api",
        "version": "1.0.0"
    }
